ui/src/app/models/dagRunDetail.model.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export class DagRunDetailModel {
+  jobRuns: JobRunModel[]
+}
+
+export class JobRunModel {
+  id: number;
+  jobName: string;
+  jobType: Something;
+  created: Date;
+  updated: Date;
+  jobStatus: Something;
+}
+
+export class Something {
```

Comment:
Please don't call it `Something` :) I guess it should be `JobStatus`

Reply:
Done :D